### ISSUE TITLE
Support `osx-arm64` in Vezel.Celerity.Runtime.Native

### ISSUE THREAD

**Issue by alexrp** [vezel-dev/celerity#101]
https://github.com/vezel-dev/celerity/blob/16d9fb38b632ff21798d8b2b2abda785edd5ed36/src/runtime/native/native.cproj#L12-L21

We need to update to a newer Vezel.Zig.Sdk version, which in turn pulls in a newer Zig toolset version. Unfortunately, doing so is blocked on ziglang/zig#15398. Also to a lesser extent ziglang/zig#13385, but musl support is significantly lower priority than macOS support. So, basically, Zig 0.12.0 waiting room.

**Comment by alexrp** [comment]
OK, found a way to convince MSBuild to treat these as warnings rather than promoting them to errors.
